### ISSUE TITLE
Could not find com.uphyca:stetho_realm:2.1.0

### ISSUE THREAD

**Issue by jovet-github-acn** [uPhyca/stetho-realm#74]
Could not determine the dependencies of task ':app:compilePoarDebugJavaWithJavac'.
> Could not resolve all task dependencies for configuration ':app:poarDebugCompileClasspath'.
   > Could not find com.uphyca:stetho_realm:2.1.0.
     Required by:
         project :app

Possible solution:
 - Declare repository providing the artifact, see the documentation at https://docs.gradle.org/current/userguide/declaring_repositories.html
 
 
My Configuration:
app/gradle
debugImplementation 'com.facebook.stetho:stetho:1.5.0'
debugImplementation 'com.uphyca:stetho_realm:2.1.0'

project gradle
maven {
            url 'https://github.com/uPhyca/stetho-realm/raw/master/maven-repo'
}

**Comment by cervebar** [comment]
Same issue with Android, Gradle and stetho_realm:2.2.2. (ಥ﹏ಥ)

```
Could not determine the dependencies of task ':app:transformClassesAndResourcesWithR8ForRelease'.
> Could not resolve all task dependencies for configuration ':app:releaseCompileClasspath'.
   > Could not find com.uphyca:stetho_realm:2.2.2.
```

**Comment by jovet-github-acn** [comment]
yes me too I used also version 2.2.2 but the same error encountered.

**Comment by DanManDKo** [comment]
Any updates?

**Comment by cervebar** [comment]
We have found a workaround for our use case (stetho and realm):
We got the `stetho_realm-2.2.2.jar`  from cached jars and linked it manually.

**app/build.gradle:** 
Don't forget to also link your folder with the jar file (we have it in `app/libs/`):

`
dependencies {
    ...
    implementation fileTree(dir: 'libs', include: ['*.jar'])
    // Stetho
    implementation "com.facebook.stetho:stetho:$stetho_version"
    // DELETE this line: implementation "com.uphyca:stetho_realm:$stetho_realm_version"
    // stetho-realm is imported via .jar file in app/libs
    ...
   }
`

**Comment by ssathishppm** [comment]
Any update team me too facing the same
